### PR TITLE
Course Listing - fix image rendering

### PR DESCRIPTION
**Pull request by jleong01**
updated to change rendering of image from thumbnail size to uploaded image size

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt [hunk 1]
```diff
@@ -7,3 +7,8 @@ taylorsuniversity-featurecarousel.js
 taylorsuniversity-accordion.js
 taylorsuniversity-herobanner.js
 taylorsuniversity-anchortabcontainer.js
+<<<<<<< HEAD
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: Remove this.

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt [hunk 2]
```diff
@@ -7,3 +7,8 @@ taylorsuniversity-featurecarousel.js
...
+taylorsuniversity-searchcourses.js
+taylorsuniversity-courselisting.js
+=======
+>>>>>>> develop
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: Remove this.

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js [hunk 3]
```diff
@@ -5,5 +5,6 @@ $(function() {
     });
 
     $('.t-accordion [data-control]').trigger('accordion.refresh');
+
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js]: this file doesn't have any change. Please remove it from this PR.

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js]: file removed

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt (hunk 2 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: line of code removed

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt (hunk 1 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: line of code removed

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js [hunk 4]
```diff
@@ -1,9 +0,0 @@
-$(function() {
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js]: If this PR will be merged then all of these lines get removed from this file. Please remove this file from this PR. If you are unable to do so. Then please put it's original content back.

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js/taylorsuniversity-accordion.js]: original content added as unable to remove file from PR

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt [hunk 5]
```diff
@@ -7,4 +7,7 @@ taylorsuniversity-featurecarousel.js
 taylorsuniversity-accordion.js
 taylorsuniversity-herobanner.js
 taylorsuniversity-anchortabcontainer.js
+taylorsuniversity-searchcourses.js
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: Please remove "taylorsuniversity-searchcourses.js". This file no longer exists.

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/taylorsuniversity/main/js.txt]: searchcourses.js line removed from file

**Pull request merged**